### PR TITLE
Disable `match nuke` for enterprise accounts

### PR DESCRIPTION
**Pull request by KrauseFx**
Also: Avoid asking for app identifier for `match nuke`

### CODE REVIEW

File: match/lib/match/nuke.rb [hunk 1]
```diff
@@ -13,7 +13,7 @@ def run(params, type: nil)
 
       params[:workspace] = GitHelper.clone(params[:git_url], params[:shallow_clone], skip_docs: params[:skip_docs], branch: params[:git_branch])
 
-      had_app_identifier = self.params[:app_identifier]
+      had_app_identifier = self.params.fetch(:app_identifier, ask: false)
```
Comment (lacostej) [inline match/lib/match/nuke.rb]: is this related to the change?

Comment (hjanuschka) [inline match/lib/match/nuke.rb]: i think its intended to disable the auto ask for the user. so in case `app_identifier` is not set - set `had_app_identifier` to nil instead of prompting user to set a app_identifier

**Review by hjanuschka — APPROVED** [review]

File: match/lib/match/nuke.rb (hunk 1 above)
Comment (hjanuschka) [inline match/lib/match/nuke.rb]: the pr body states that this is to avoid asking for app_identifier
